### ISSUE TITLE
Add lint preventing EIP references in backticks

### ISSUE THREAD

**Issue by SamWilsn** [ethereum/eipw#90]
Many authors attempt to circumvent the EIP link requirements by putting EIP-XXXX in backticks (eg. \`EIP-1234\`.) This is bad form, and should be rejected.

**Comment by VictoriaAde** [comment]
@SamWilsn I'd like to work on this.
Can I be assigned?

**Comment by VictoriaAde** [comment]
Noted!

**Comment by Hari-Bombon** [comment]
@SamWilsn I would love to work on this repo. Could you please assign me?

**Comment by SamWilsn** [comment]
@Hari-Bombon we should give the user above a chance to work on this, since she asked first. #11 or #5 might be good candidates to try instead?

**Comment by VictoriaAde** [comment]
Hi @SamWilsn, I have added the lint rule but my test keeps failing.
What do you think I'm doing wrong?

![image](https://github.com/ethereum/eipw/assets/60205816/a4ec0dc9-3970-4864-847b-66ee7455f022)

**Comment by SamWilsn** [comment]
@VictoriaAde I'd need to see a bit more of your code (do you have a branch somewhere?) but if I had to guess, it looks like your `src` variable isn't a correctly formatted EIP document. It's missing the `---` somewhere.

**Comment by VictoriaAde** [comment]
@SamWilsn I looked at what you said and now if I run cargo run or test, it will giving the below message.
`<SOURCE>...
` I can't seem to figure out what the issue is.
 
 
![image](https://github.com/ethereum/eipw/assets/60205816/7aaee75f-0bbb-4f8f-a5cd-54fe08ad964d)

**Comment by SamWilsn** [comment]
Right, forgot about that. The project is divided into a few crates (aka packages):

 * [`eipw`](https://github.com/ethereum/eipw/blob/master/Cargo.toml) is at the root of the repository, and provides a command-line interface to the tool.
 * [`eipw-preamble`](https://github.com/ethereum/eipw/blob/master/eipw-preamble/Cargo.toml) is a library that provides functions for parsing the preamble of EIPs (the stuff at the beginning of proposals between `---` lines.)
 * [`eipw-lint`](https://github.com/ethereum/eipw/blob/master/eipw-lint/Cargo.toml) is where the majority of the interesting code lives, including the lints themselves.
 * [`eipw-lint-js`](https://github.com/ethereum/eipw/blob/master/eipw-lint-js/Cargo.toml) is the glue that connects to JavaScript, and will likely be uninteresting for this issue.

To run the tests for lints, you'll need to change directory into `eipw-lint`, and run the tests there:

```bash
cd eipw-lint
cargo test
```

Alternatively, you can run _all_ the tests from the root directory:

```bash
cargo test --all
```

---

Regarding the `<SOURCES>...` message, the tool is asking for the file you'd like to check. For example:

```bash
cargo run -- ~/EIPs/EIPS/eip-3074.md
```

**Comment by VictoriaAde** [comment]
Oh my, thank you so much.
I will try these out.

**Comment by VictoriaAde** [comment]
@SamWilsn, I made a PR, please check it out
